app/test/frontend/golden/authorized_page.html

```diff
@@ -24,7 +24,7 @@
   <meta property="og:description" content="Pub is the package manager for the Dart programming language, containing reusable libraries &amp; packages for Flutter, AngularDart, and general Dart programs." />
   <link rel="alternate" type="application/atom+xml" title="Updated Packages Feed for Pub" href="/feed.atom" />
   <link href="/static/highlight/github.css" rel="stylesheet" />
-  <link href="/static/css/style.css?hash=18js5k8h2d638mkc6lerk5gr5g5hjavp" rel="stylesheet" type="text/css" />
+  <link href="/static/css/style.css?hash=cc69ncs8qti1ldoshp4tnusgh051fi9j" rel="stylesheet" type="text/css" />
```

Comment:
Why is this updated like this? Could a http style etag or something work instead?

Reply:
We have both: ETag support (+ IfModifiedSince support), and an additional hash in the URL. The later is used because that way we can use a really high value for browser-side cache timeout, otherwise we would have a window where we are already serving the new version with e.g. a new layout, but the browser may not even attempt to request the resource from the server. It is used only for the two critical resource for use: the main CSS and the main JS file.